scripts/prepare-swagger-push.sh

```diff
@@ -23,6 +23,12 @@ if [[ ! -f "${SWAGGER_JSON_FILE}" ]]; then
     exit 1
 fi
 
+# Set host URL based on environment, defaulting to production
+host_url="evergreen.mongodb.com"
```

Comment:
Should we default to staging and then override when the environment is production? I'm not strong about this and we can keep as-is since it doesn't affect the end location- but we have many staging projects but only one prod project

Reply:
I'm okay with that. It's worth noting that individual stagings don't track (at least by default) the evergreen repo and they don't have access to the same bucket (mciuploads) I think, so we are good on that avenue